test/terraform_aws_acm_cert_test.go

```diff
@@ -0,0 +1,58 @@
+package test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestACMCertCreation(t *testing.T) {
+	t.Parallel()
+
+	tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "../", "examples/simple")
+
+	testName := fmt.Sprintf("terratest-%s", strings.ToLower(random.UniqueId()))
+	awsRegion := "us-west-2"
+	zoneName := "truss.coffee"
+	domainName := "infra-test.truss.coffee"
+	acmDomain := fmt.Sprintf("%s.%s", testName, domainName)
+	//"${var.test_name}.${local.zone_name}"
+	//zoneName := "truss.coffee"
+
+	terraformOptions := &terraform.Options{
+		// The path to where our Terraform code is located
+		TerraformDir: tempTestFolder,
+
+		// Variables to pass to our Terraform code using -var options
+		Vars: map[string]interface{}{
+			"test_name":   testName,
+			"region":      awsRegion,
+			"domain_name": domainName,
+			"environment": testName,
```

Comment:
```suggestion
```

Reply:
Also, in general, I'd leave this just a string reflecting the actual env (i.e. test).

Reply:
Since I defined the local var, like your other feedback, I simply got rid of this. I hope that's ok?

Reply:
Yep. That was for the future :)